documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -198,7 +219,17 @@ $ sudo cp arch/arm/boot/dts/overlays/README /boot/firmware/overlays/
 $ sudo cp arch/arm/boot/zImage /boot/firmware/$KERNEL.img
 ----
 
-TIP: On multi-core Raspberry Pi models, the `-j <n>` option distributes work between cores. This can speed up compilation significantly. Run `nproc` to see how many processors you have; we recommend passing a number 1.5x your number of processors.
+[TIP]
+====
+Alternatively, copy the kernel with a different filename (e.g. `kernel-myconfig.img`) instead of overwriting the `kernel.img` file. Then, edit `config.txt` in the boot partition to select your kernel:
+
+[source,ini]
+----
+kernel=kernel-myconfig.img
+----
+
+This has the advantage of keeping your custom kernel separate from the stock kernel image managed by the system. With this arrangement, you can quickly revert to a stock kernel in the event that your kernel cannot boot.
```

Comment:
"With this arrangement (in combination with also using LOCALVERSION), ..." :slightly_smiling_face: